Intersect.Server/Web/RestApi/Types/Chat/ChatMessageResponseBody.cs

```diff
@@ -0,0 +1,8 @@
+namespace Intersect.Server.Web.RestApi.Types.Chat;
+
+public class ChatMessageResponseBody(bool success, ChatMessage chatMessage)
```

Comment:
make this into a record (not record struct, just record) and only use a primary constructor don't explicitly list the parameters